app/views/components/gds/paragraph.scala.html

```diff
@@ -0,0 +1,21 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this()
+
+@(html: Html, classes: String = "govuk-body", id: Option[String] = None)
+
+<p class="@classes" @id.map{id => id="@id" }>@html</p>
```

Comment:
Nit: might be worth updating the original paragraphBody template to call this new paragraph template to avoid duplication of the definition of the '<p>' tag

Reply:
Tbh... the duplication of tags is less worrying to me than the nesting of partials, which makes more problematic using them in pure scala-code files, due to the dependency injection requirement.